tests/model_serving/model_server/utils.py

```
@@ -217,3 +225,74 @@
+            chat_query=chat_query,
+            completion_query=completion_query,
+        )
+
+    start_time = time.time()
+    with concurrent.futures.ThreadPoolExecutor(max_workers=num_concurrent) as executor:
+        while time.time() - start_time < duration:
+            futures = [executor.submit(make_request) for _ in range(num_concurrent)]
+            concurrent.futures.wait(fs=futures)
+
+
+def test_ovms_concurrent_load(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)